### PR TITLE
Add Menu Navbar

### LINKED ISSUE #7: Continous Integration and Deployment
1. Write a few unit tests
2. Bring in Travis CI or similar application
3. Setup automatic deployment

### PR DESCRIPTION
**Pull request by maryamyasaei**
Resolve #7 issue

### CODE REVIEW

File: templates/base/navbar.html [hunk 1]
```diff
@@ -8,7 +8,31 @@
...
+          <a class="dropdown-item" href="#">Another action</a>
+          <div class="dropdown-divider"></div>
+          <a class="dropdown-item" href="#">Something else here</a>
+        </div>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link disabled" href="#">Disabled</a>
+      </li>
+    </ul>
+    <form class="form-inline my-2 my-lg-0">
```
Comment (hesamd) [inline templates/base/navbar.html]: The task doesn't require a form please remove the search box.

File: templates/base/navbar.html [hunk 2]
```diff
@@ -8,7 +8,31 @@
     <ul class="navbar-nav mr-auto">
       <li class="nav-item active">
         <a class="nav-link" href="/">Home <span class="sr-only">(current)</span></a>
+      </li><li class="nav-item">
+        <a class="nav-link" href="#">Link</a>
       </li>
+      <li class="nav-item dropdown">
```
Comment (hesamd) [inline templates/base/navbar.html]: please move the drop down to the left. Check Bootstrap syntax how to position items to the left and right.

File: templates/base/navbar.html [hunk 3]
```diff
@@ -8,7 +8,31 @@
...
+        <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+          Dropdown
+        </a>
+        <div class="dropdown-menu" aria-labelledby="navbarDropdown">
```
Comment (hesamd) [inline templates/base/navbar.html]: Use proper Django links here, you need to have Register, Login and Logout. Check the syntax in `templates/accounts/....`

**Review by hesamd — COMMENTED** [review]
Please always upload a screenshot of your changes if you have a visible change

File: templates/base/navbar.html (hunk 1 above)
Comment (maryamyasaei) [reply templates/base/navbar.html]: done..

File: templates/base/navbar.html [hunk 4]
```diff
@@ -1,14 +1,28 @@
...
+                  </a>
+                  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+                      <a class="dropdown-item" href="{{ login_url }}">sign in</a>
+                       <a class="dropdown-item"  href="{{ signup_url }}">Register</a>
+                  </div>
+                </li>
+              </ul>
+            </div>
+        </div>
+      </nav>
```
Comment (hesamd) [inline templates/base/navbar.html]: please don't forget to add a new line at the end of your files (see the red icon above)

File: templates/base/navbar.html [hunk 5]
```diff
@@ -1,14 +1,28 @@
...
+          <ul class="navbar-nav mr-auto">
+            <li class="nav-item active">
+              <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
+            </li>
+          </ul>
+            <div class="navbar-collapse"  collapse w-100 order-3 dual-collapse2">
+              <ul class="navbar-nav ml-auto">
+                <li class="nav-item dropdown">
+                  <a class="nav-link dropdown-toggle" href="#'>  id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                    Sign In
```
Comment (hesamd) [inline templates/base/navbar.html]: Rename Sign In to Account, also capitalize the other sign in the dropdown

**Review by hesamd — DISMISSED** [review]

**Review by hesamd — APPROVED** [review]

**Pull request merged**